CDTDatastore/HTTP/CDTSessionCookieInterceptorBase.m

```diff
@@ -74,11 +74,30 @@ - (BOOL)hasValidCookieWithName:(nonnull NSString*)cookieName forRequestURL:(nonn
  */
 - (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorContext *)context
 {
-    if (context.response.statusCode == 401 && self.shouldMakeSessionRequest) {
+    bool retryAndAttemptNewSession = NO;
+    if (self.shouldMakeSessionRequest) {
+        if (context.response.statusCode == 401) {
+            retryAndAttemptNewSession = YES;
+        } else if (context.response.statusCode == 403) {
```

Comment:
@tomblench did you think this block should move down into the `CDTSessionCookieInterceptor`?